### ISSUE TITLE
New steam vr update.

### ISSUE THREAD

**Issue by Ho0ty** [matzman666/OpenVR-InputEmulator#121]
several options in the openvr input emulator are no longer available due to a recent steam update. A big update is probably in store for you. Good luck. 

Edit: i have an HTC vive.

**Comment by joemarshall** [comment]
I think this is probably because the vive driver is now using the IVRDriverInput_002 input API which turned up in the SDK 2 months ago. It will need an updated hook in openvr-inputemulator.

If so, it isn't a massive code change to fix it, but I don't have time and access to my vive to do it in the short term. I don't know if this project is still active, as there's no response on issues for some time.

**Comment by joemarshall** [comment]
Hey,

I had a bit of time to patch my openvrinput emulator so that it hooks either of 
IVRDriverInput_001 or IVRDriverInput_002 which means it works with vive again, at least for what I'm wanting to do (send axis actions).

The attached is an updated driver which will work for old or new versions. I haven't got time right now to upload the patch to github, sorry.

You need to stick the dll file in this zip into `C:\Program Files (x86)\Steam\steamapps\common\SteamVR\drivers\00vrinputemulator\bin\win64 ` replacing the one that is already there.

[driver_00vrinputemulator.zip](https://github.com/matzman666/OpenVR-InputEmulator/files/2419392/driver_00vrinputemulator.zip)

**Comment by Viperlotus** [comment]
Hello everyone,

I wanted to join in this discussion because I was also relying on OpenVR Input Emulator before it was broken by Steam updates, as I needed to remap the system buttons of my Vive wands.  I like to bind my left wand's system button to the keyboard key I use for push-to-talk in external VOIP software like Discord or Teamspeak (I have to set it as a toggle though, since holding down a system button for too long shuts off the respective wand).

Specifically, I wanted to thank joemarshall for the file he attached to the post above mine.  Even though I still can't access most of the options I was able to before the program was broken by Steam updates after I installed your file, I was able to make my old remap profile work by selecting and applying it.

Thank you again for that fix, however small and makeshift it may be.  I hope you'll be able to properly complete and upload the patch in the future.

**Comment by Johnyknowhow** [comment]
Yeah, unfortunately, remaps don't seem to be available in the menus as of now. Joemarshall's driver fix restored some functionality but the remapping menu is still almost entirely gone. 

Sucks that the dev is MIA, especially since I just discovered this utility and it looks perfect for what I need to do.

**Comment by AGR-13** [comment]
I'm missing the function of the motion compensation. I need it for my simulation chair. Without it - it's not possible to use. 
I tried the fix from joemarshall (thx anyway for trying to help) but unfortunately it didn't work at my installation. I have a VIVE Pro.
